jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/ImageNameTest.java

```diff
@@ -123,6 +123,22 @@ private void verifyData(Object[] data) {
     @Test
     void testIllegalFormat() {
         assertThrows(IllegalArgumentException.class, () -> new ImageName(""));
+
+        // New test for too long repository name
+        String tooLongName = generateTooLongImageName();
+        assertThatIllegalArgumentException()
```

Comment:
Sorry, I meant to add a new test; not modify existing test. Could you please move this code block into a separate test?

```java
@Test 
void shouldThrowExceptionOnTooLongImageName() {

}
```

Reply:
> Sorry, I meant to add a new test; not modify existing test. Could you please move this code block into a separate test?
> 
> ```java
> @Test 
> void shouldThrowExceptionOnTooLongImageName() {
> 
> }
> ```

Hii @rohanKanojia did all the changes you told me do, but getting a build failure

[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:38 min
[INFO] Finished at: 2024-01-22T14:44:48Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.1.2:test (default-test) on project jkube-kit-remote-dev: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/jenkins/agent/workspace/Build_Java8_PR-2555/jkube-kit/remote-dev/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.1.2:test (default-test) on project jkube-kit-remote-dev: There are test failures.

is it due to my changes? or can you suggest how can i fix this?

Reply:
> > Sorry, I meant to add a new test; not modify existing test. Could you please move this code block into a separate test?
> > ```java
> > @Test 
> > void shouldThrowExceptionOnTooLongImageName() {
> > 
> > }
> > ```
> 
> Hii @rohanKanojia did all the changes you told me do, but getting a build failure
> 
> [INFO] BUILD FAILURE [INFO] ------------------------------------------------------------------------ [INFO] Total time: 03:38 min [INFO] Finished at: 2024-01-22T14:44:48Z [INFO] ------------------------------------------------------------------------ [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.1.2:test (default-test) on project jkube-kit-remote-dev: There are test failures. [ERROR] [ERROR] Please refer to /home/jenkins/agent/workspace/Build_Java8_PR-2555/jkube-kit/remote-dev/target/surefire-reports for the individual test results. [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream. [ERROR] -> [Help 1] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.1.2:test (default-test) on project jkube-kit-remote-dev: There are test failures.
> 
> is it due to my changes? or can you suggest how can i fix this?

test added by me is passing

Reply:
@Devashishbasu : This failure seems unrelated to your changes. Probably it's a flaky test that needs to be fixed. Could you please try pushing again and see if you see the issue again?

Reply:
> @Devashishbasu : This failure seems unrelated to your changes. Probably it's a flaky test that needs to be fixed. Could you please try pushing again and see if you see the issue again?

Hi @rohanKanojia i've pushed the code again and seeing the same issue again...it is the same issue which you have opened  #2576

Reply:
> @Devashishbasu : This failure seems unrelated to your changes. Probably it's a flaky test that needs to be fixed. Could you please try pushing again and see if you see the issue again?

Hii @rohanKanojia ...still the same issue persists